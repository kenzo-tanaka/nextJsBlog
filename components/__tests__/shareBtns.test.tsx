import React from "react";
import { createRenderer } from "react-test-renderer/shallow";
import ShareBtns from "../shareBtns";

const renderer = createRenderer();

test("ShareBtns", () => {
  renderer.render(<ShareBtns slug="test-slug" title="test-title" />);
  const renderedOutput = renderer.getRenderOutput();
  expect(renderedOutput).toMatchSnapshot();
});
