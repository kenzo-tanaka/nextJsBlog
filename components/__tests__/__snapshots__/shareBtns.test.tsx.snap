// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShareBtns 1`] = `
<div
  className="btnsWrapper mt-5 text-center"
>
  <a
    className="p-1 text-center no-underline inline-block text-white truncate align-middle font-bold bg-black text-xl w-10 h-10 leading-10 rounded-full"
    href="https://twitter.com/share?url=http://localhost:3000/test-slug&text=test-title%20%7C%20Kenzo%20Blog&via=kenzoooooB"
    rel="nofollow"
    target="_blank"
  >
    <img
      alt="Twitter"
      src="/images/twitter-white.svg"
    />
  </a>
  <a
    className="mx-3 text-center no-underline inline-block text-white truncate align-middle font-bold bg-black text-xl w-10 h-10 leading-10 rounded-full"
    href="https://www.facebook.com/share.php?u=http://localhost:3000/test-slug"
    rel="nofollow"
    target="_blank"
  >
    F
  </a>
  <a
    className="text-center no-underline inline-block text-white truncate align-middle font-bold bg-black text-xl w-10 h-10 leading-10 rounded-full"
    href="http://b.hatena.ne.jp/add?mode=confirm&url=http://localhost:3000/test-slug"
    rel="nofollow"
    target="_blank"
  >
    B!
  </a>
</div>
`;
